e2e/shared/testdata/record_v031.json

```diff
@@ -54,5 +56,16 @@
       }
     }
   ],
-  "signature": {}
+  "signature": {
+    "algorithm": "ES256",
+    "certificate": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0t",
+    "content_bundle": "eyJ0ZXN0IjogInZhbHVlIn0=",
+    "content_type": "application/json",
+    "signature": "MEUCIQDTest123Signature456789",
+    "signed_at": "2025-09-11T10:00:00Z",
+    "annotations": {
+      "signer": "test-authority",
+      "purpose": "testing"
+    }
+  }
```

Comment:
does the validation require signature? on the directory side the signature is a referrer external to the record